src/ConstructiveSolidGeometry/IO.jl

```diff
@@ -22,8 +22,10 @@ end
 
 
 function parse_CSG_transformation(::Type{T}, dict::AbstractDict, input_units::NamedTuple) where {T}
-    r = if haskey(dict, "rotation")
-        parse_rotation_vector(T, dict["rotation"], input_units.length)
+    r = if haskey(dict, "rotate")
+        SMatrix{3, 3, Float32, 9}(parse_rotation_matrix(T, dict["rotate"], input_units.length))
```

Comment:
Ẁhy `Float32`? To avoid rounding errors?

Reply:
Thanks... I am in a hurry right now :grimacing: